Makefile

```diff
@@ -73,7 +73,7 @@ clean_all: clean clean_deps clean_vendor
 
 .PHONY: format_api_docs_links
 format_api_docs_links:
-	echo $(DOCS_FILES) | xargs perl -p -i -e 's@\bhttps?://crystal-lang\.org/api/(?!toplevel|[A-Z])[^ )/]+/([^ )]+\.html\b)@https://crystal-lang.org/api/\1@g'
+	echo $(DOCS_FILES) | xargs sed -i -E -e 's@\bhttps?://(www\.)?crystal-lang\.org/api/([0-9]+(\.[0-9]+)+|latest|master)/([^ )]+\.html)\b@https://crystal-lang.org/api/\4@g'
```

Comment:
I believe the regex won't match paths that contain a `/` in the path after the version prefix (such as `/api/latest/Foo/Bar.html`).

Reply:
Hm no, it definitely catches them.
There's the version, then 1 slash, then anything except space or parenthesis. That "anything" includes slashes

Reply:
Anyway I made sure to try it out on the site

Reply:
Ah totally right. I was confused by `[^ )/]` in the original code.